portability-gateway/src/main/java/org/dataportabilityproject/gateway/action/startjob/StartJobAction.java

```diff
@@ -179,24 +185,41 @@ private void encryptAndUpdateJobWithCredentials(
       PortabilityJob job,
       String encryptedExportAuthCredential,
       String encryptedImportAuthCredential) {
-    PublicKey publicKey =
-        asymmetricKeyGenerator.parse(
-            BaseEncoding.base64Url().decode(job.jobAuthorization().encodedPublicKey()));
+
 
     // Encrypt the data with the assigned workers PublicKey and persist
-    // NOTE: These auth datas are already encrypted with the session symmetric encryption key
-    Encrypter crypter = EncrypterFactory.create(publicKey);
-    String encryptedExportAuthData = crypter.encrypt(encryptedExportAuthCredential);
-    logger.debug("Encrypted export auth data, has length: {}", encryptedExportAuthData.length());
-    String encryptedImportAuthData = crypter.encrypt(encryptedImportAuthCredential);
-    logger.debug("Encrypted import auth data, has length: {}", encryptedImportAuthData.length());
+
+    /*
+     * NOTE: These auth datas are already encrypted with the session symmetric encryption key
```

Comment:
I removed wordy explanation, hopefully this is clearer.

Background:
I made the mistake of first attempting to encrypt all the data with RSA/asymmetric key. This is wrong because RSA is mean to only encrypt a fixed amount of information, not arbitrarily long data.

The general way to do this is to encrypt the arbitrarily long data (in our case, auth data) with a symmetric key then encrypt the symmetric key itself with RSA public key since it's a smaller length.  That way only the private key can unpack the symmetric key to decrypt the data.